packages/scripts/src/remove-cases.js

```diff
@@ -384,7 +389,20 @@ const getReferencesPrefixedWith = async (startsWith) => {
  * @returns {Promise<void>}
  */
 export const removeAllTrainingCases = async () => {
-	// const references = await getReferencesPrefixedWith('TRAIN0110004');
 	const references = await getReferencesPrefixedWith('TRAIN');
```

Comment:
likewise we could use the sector Training here too?

Reply:
Discussed and resolved